src/macros.rs

```diff
@@ -1219,7 +1219,17 @@ impl MacroBranch {
         }
 
         // 5 = " => {"
-        let mut result = format_macro_args(context, self.args.clone(), shape.sub_width(5)?)?;
```

Comment:
can we remove this leading comment since we comment what 5 and 6 represent down bellow

Reply:
Removed